autoscheduler/frontend/src/components/SchedulingPage/Schedule/MeetingCard/MeetingCard.tsx

```diff
@@ -3,32 +3,157 @@ import { Typography } from '@material-ui/core';
 import * as styles from './MeetingCard.css';
 import Meeting, { MeetingType } from '../../../../types/Meeting';
 import ScheduleCard from '../ScheduleCard/ScheduleCard';
+import meetingTimeText from '../../../../utils/meetingTimeText';
+import { meetingBuildingWithRoom } from '../../../../utils/meetingBuilding';
+
+enum MeetingCardSizes {
```

Comment:
Small but typescript enum names are singular, not plural since it makes enum member access look more natural (sizes.large vs. size.large)
```suggestion
enum MeetingCardSize {
```